unix-ffi/json/manifest.py

```diff
@@ -0,0 +1,4 @@
+metadata(version="0.2.0")
+
+require("re", unix_ffi=True)
```

Comment:
The `unix_ffi` argument is no longer supported.

(I wonder how the CI even passes on this PR??)

Reply:
CI passes because `unix_ffi` is swallowed by the `**kwargs` and ignored.